src/Components/CreateImageWizard/steps/Users/component/UserInfo.tsx

```diff
@@ -32,6 +37,14 @@ const UserInfo = () => {
     dispatch(setUserPasswordByIndex({ index: index, password: value }));
   };
 
+  const handleSshKeyChange = (
+    _event: React.FormEvent<HTMLInputElement>,
+    value: string
+  ) => {
+    const index = 0;
```

Comment:
Why are we setting the index here? Shouldn't we work with the "global" index defined under dispatch?